gcloud/credentials.py

```diff
@@ -72,8 +73,34 @@ def get_credentials():
     return client.GoogleCredentials.get_application_default()
 
 
+def get_for_service_account_json(private_key_path, scope=None):
```

Comment:
1) This isn't for PKCS12 keys, its for JSON keys.
2) This is supposed to be a path, not a credentials object. It returns a credentials object.